### PR TITLE
Some small changes

### PR DESCRIPTION
**Pull request by pirapira**
This pull-request is a mixture of
* a change in a comment
* some refactoring of common code.

### CODE REVIEW

**Comment by CJentzsch** [comment]
please rebase
can't merge

**Comment by pirapira** [comment]
rebased.

File: DAO.sol [hunk 1]
```diff
@@ -276,15 +276,13 @@ contract DAO is DAOInterface, Token, Crowdfunding {
         // burn tokens
         uint tokenToBeBurned = (balances[msg.sender] * p.splitBalance) / (totalSupply + rewards);
         if (balances[msg.sender] < tokenToBeBurned) { // This happens when the DAO has had incomes not counted in `rewards`.
-            totalSupply -= balances[msg.sender];
-            balances[msg.sender] = 0;
-        } else {
-            totalSupply -= tokenToBeBurned;
-            balances[msg.sender] -= tokenToBeBurned;
+            tokenToBeBurned = balances[msg.sender];
```
Comment (CJentzsch) [inline DAO.sol]: this seems to be wrong. I only want to move the funds and burn that share of the tokens.
with the remaining tokens I calculate the right to receive future rewards, and then I set them to zero.

**Comment by pirapira** [comment]
I removed the problematic commit.  I think the original code has the same problem as you pointed out.

**Pull request merged**